pkg/apis/pipeline/v1alpha1/task_types.go

```diff
@@ -68,8 +68,7 @@ type TaskSpec struct {
 
 // TaskStatus defines the observed state of Task
 type TaskStatus struct {
-	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
-	// Important: Run "make" to regenerate code after modifying this file
+	// Important: Run "./hack/update-codegen.sh" to regenerate code after modifying this file
```

Comment:
🙏 thank you!

Reply:
p.s. really appreciate your attention to detail cleaning up lots of comments and docs @vdemeester !!! ❤️